test/functional/Java8andUp/src_80/org/openj9/test/java/lang/Test_Thread_Java8to10.java

```diff
@@ -0,0 +1,278 @@
+			if (d >= 0)
+				delay = d;
+		}
+	}
+
+	static class ResSupThread implements Runnable {
+		Thread parent;
+		volatile int checkVal = -1;
+
+		public void run() {
+			try {
+				synchronized (this) {
+					this.notify();
+				}
+				while (true) {
+					checkVal++;
+					zz();
+					Thread.sleep(100);
+				}
+			} catch (InterruptedException e) {
+				return;
+			} catch (BogusException e) {
+				try {
+					// Give parent a chance to sleep
+					Thread.sleep(500);
+				} catch (InterruptedException x) {
+				}
+				parent.interrupt();
+				while (!Thread.currentThread().isInterrupted()) {
+					// Don't hog the CPU
+					try {
+						Thread.sleep(50);
+					} catch (InterruptedException x) {
+						// This is what we've been waiting for...don't throw it
+						// away!
+						break;
+					}
+				}
+			}
+		}
+
+		public void zz() throws BogusException {
+		}
+
+		public ResSupThread(Thread t) {
+			parent = t;
+		}
+
+		public synchronized int getCheckVal() {
+			return checkVal;
+		}
+	}
+
+	static class BogusException extends Throwable {
+		public BogusException(String s) {
+			super(s);
+		}
+	}
+
+	Thread st, ct, spinner;
+
+	static boolean calledMySecurityManager = false;
+
+	// used in test_defaultMemoryArea to pass status from thread started
+	// back to thread running test case
+	boolean pass = true;
+
+	/**
+	 * @tests java.lang.Thread#destroy()
+	 */
+	@Test
+	public void test_destroy() {
+		try {
+			new Thread().destroy();
+			Assert.fail("NoSuchMethodError expected");
+		} catch (NoSuchMethodError e) {
+			// expected
+		}
+	}
+
+	/**
+	 * @tests java.lang.Thread#stop(java.lang.Throwable)
+	 */
+	@Test
+	public void test_stop3() {
+		/* [PR CMVC 94728] AccessControlException on dead Thread */
+		Thread t = new Thread("t");
+		class MySecurityManager extends SecurityManager {
+			public boolean intest = false;
+			public boolean checkAccess = false;
+
+			public void checkAccess(Thread t) {
+				if (intest) {
+					checkAccess = true;
+				}
+			}
+		}
+		MySecurityManager sm = new MySecurityManager();
+		System.setSecurityManager(sm);
+		try {
+			sm.intest = true;
+			try {
+				t.stop(new ThreadDeath());
+			} catch (UnsupportedOperationException e) {
+				// Expected
+			} catch (Throwable e) {
+				Assert.fail("unexpected exception: " + e);
+			}
+			sm.intest = false;
+			AssertJUnit.assertFalse("checkAccess 1", sm.checkAccess);
+			t.start();
+			try {
+				t.join(2000);
+			} catch (InterruptedException e) {
+			}
+			sm.intest = true;
+			sm.checkAccess = false;
+			try {
+				t.stop(new ThreadDeath());
+			} catch (UnsupportedOperationException e) {
+				// Expected
+			} catch (Throwable e) {
+				Assert.fail("unexpected exception: " + e);
+			}
+			AssertJUnit.assertFalse("checkAccess 2", sm.checkAccess);
+			sm.intest = false;
+		} finally {
+			System.setSecurityManager(null);
+		}
+	}
+
+	/**
+	 * @tests java.lang.Thread#stop(java.lang.Throwable)
+	 */
+	@Test
+	public void test_stop4() {
```

Comment:
I don't think we need this test, test_stop6() is sufficient. test_stop4() seems like a modified version of the test that used to validate stop(Throwable) but now doesn't really test anything. Removing this test also obsoletes the ResSupThread class, the BogusException class, and the st variable.